Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py

```diff
@@ -338,11 +323,24 @@ def wildfire_upload_file(upload):
     return result, upload_file_data
 
 
-def wildfire_upload_file_command():
-    uploads = argToList(demisto.args().get('upload'))
+def wildfire_upload_file_with_polling_command(args):
+    return run_polling_command(args, 'wildfire-upload', wildfire_upload_file_command,
+                               wildfire_get_report_command, 'FILE')
+
+
+def wildfire_upload_file_command(args):
+    assert_upload_argument(args)
+    uploads = argToList(args.get('upload'))
+    command_results_list = []
     for upload in uploads:
-        result, upload_file_data = wildfire_upload_file(upload)
-        create_upload_entry(upload_file_data, 'WildFire Upload File', result)
+        result, upload_body = wildfire_upload_file(upload)
+        pretty_upload_body = prettify_upload(upload_body)
+        human_readable = tableToMarkdown('WildFire Upload File', pretty_upload_body, removeNull=True)
+        command_results = (CommandResults(outputs_prefix=WILDFIRE_REPORT_DT_FILE,
+                                          outputs=pretty_upload_body, readable_output=human_readable,
+                                          raw_response=result))
+        command_results_list.append(command_results)
```

Comment:
This can be extracted to a function. The same code is used in the next functions as well.

Reply:
The upload function is different (wildfire_upload_file vs. wildfire_upload_file_url) + the human readable header is different, so extracting to a different function (with the upload function and the human readable as arguments) will cause the code to be less readable. If you think otherwise, I can extract it.

Reply:
@Noy-Maimon not critical, feel free to leave it as is.